api/api/controllers/agent_controller.py

```diff
@@ -74,7 +73,9 @@ async def delete_agents(
             'type': type,
             'version': version,
             'last_login': older_than,
-            'persistent_connection_node': node_name,
+            'is_connected': is_connected,
+            'host.ip': request.query_params.get('remote.ip', None),
+            'host.os.full': request.query_params.get('os.full', None),
```

Comment:
Why are you getting the parameters from the request and not as arguments of the controller? I'd normalize this to don't mix.

Reply:
It's not possible to get nested values from the controller's arguments. I initially tried it that way but they were returning `null` no matter what I did.

Here's some examples on other endpoints where this is the case as well 

https://github.com/wazuh/wazuh/blob/641a12b682770b840fa7218f3b38cb40cc69e1e2/api/api/controllers/agent_controller.py#L541-L543 

https://github.com/wazuh/wazuh/blob/641a12b682770b840fa7218f3b38cb40cc69e1e2/api/api/controllers/experimental_controller.py#L662-L664